src/main/java/com/google/api/codegen/InterfaceView.java

```diff
@@ -26,15 +27,13 @@
 public class InterfaceView implements InputElementView<Interface> {
 
   /**
-   * Gets the reachable interfaces of the model.
+   * Gets the interfaces for the apis in the service config.
    */
   @Override
   public Iterable<Interface> getElementIterable(Model model) {
     ArrayList<Interface> interfaces = new ArrayList<>();
-    for (Interface iface : model.getSymbolTable().getInterfaces()) {
-      if (iface.isReachable()) {
```

Comment:
Was isReachable doing any work before? And if so, do we need something equivalent in the updated code?